### ISSUE TITLE
Error with GPA fetch from Course Critique

### ISSUE THREAD

**Issue by abhitirumala** [64json/gt-scheduler#14]
When selecting a course, there is an issue with the average GPA fetch from critique.gatech.edu

![image](https://user-images.githubusercontent.com/13686695/98019390-b2322a80-1dcf-11eb-904f-a9dad666dc77.png)

> `The origin "https://jasonpark.me" has sent too many requests.
> The number of requests is limited to 200 per 60 minutes. Please self-host CORS Anywhere if you need more quota. See https://github.com/Rob--W/cors-anywhere#demo-server`

**Possible Solutions:**

- Run CORS-Anywhere Server in parallel with GT Scheduler to proxy Cross-origin requests to critque.gatech.edu

- Wait until the course critique api is functional and make requests through that

**Comment by 64json** [comment]
Yeah it seems like we need to set up a server soon. Thanks for letting me know!
